platform/darwin/src/MGLBackgroundStyleLayer.h

```diff
@@ -69,6 +69,11 @@ MGL_EXPORT
 #endif
 
 /**
+ `backgroundColor` transition attributes.
```

Comment:
> The transition affecting any changes to this layer’s `backgroundColor` property.
>
> This property corresponds to the `background-color-transition` property in the style JSON file format.

(We can't quite say "in the Mapbox Style Specification", as we do elsewhere, because of mapbox/mapbox-gl-js#4081.)